pokemongo_bot/event_handlers/logging_handler.py

```diff
@@ -4,7 +4,10 @@
 
 
 class LoggingHandler(EventHandler):
-    def handle_event(self, event, sender, level, data):
+    def handle_event(self, event, sender, level, formatted_msg, data):
```

Comment:
because we might want to have logger messages with real text instead of just parameters for events